cmd/lotus-shed/splitstore.go

```diff
@@ -353,6 +355,40 @@ func deleteSplitstoreKeys(lr repo.LockedRepo) error {
 	return nil
 }
 
+func printSplitstoreKeys(lr repo.LockedRepo) error {
+	ds, err := lr.Datastore(context.TODO(), "/metadata")
+	if err != nil {
+		return xerrors.Errorf("error opening datastore: %w", err)
+	}
+	if closer, ok := ds.(io.Closer); ok {
+		defer closer.Close() //nolint
+	}
+
+	res, err := ds.Query(context.Background(), query.Query{Prefix: "/splitstore"})
```

Comment:
@ZenGround0 can you point me to where this data is originally written? I can't find it. Or is this legacy data?

Reply:
oh, found them: https://github.com/filecoin-project/lotus/blob/0e4857a11ba57c8c29e27dc063f65da535bd5ff0/blockstore/splitstore/splitstore.go#L30-L51